composer.json

```diff
@@ -13,7 +13,9 @@
     "require": {
         "php": "^5.5.9|^7.0",
         "symfony/framework-bundle": "^2.7|3.*",
-        "symfony-cmf/resource": "^1.0"
+        "symfony/options-resolver": "^2.7|3.*",
+        "symfony-cmf/resource": "dev-admin_metadata",
+        "puli/repository": "@beta"
```

Comment:
The problem here is that both 1.1 and 1.0 aren't stable of Puli. Our version constraint in `symfony-cmf/resource` is something like `^1.0.0-betaN`. As dev versions are allowed, puli will use `1.1.0-dev` (as it's the highest dev version in the version constraint).

If we would do `~1.0.0-betaN` in `symfony-cmf/resource`, things work as expected (it'll install the 1.0 dev branch and not master).